addons/website/models/ir_http.py

```diff
@@ -371,8 +371,8 @@ def generate(self, query=None, args={}):
         query = query and query.startswith('website.') and query[8:] or query
         if query:
             domain += [('key', 'like', query)]
-        website_id = request.context.get('website_id') or request.env['website'].search([], limit=1).id
-        domain += ['|', ('website_id', '=', website_id), ('website_id', '=', False)]
+        website = request.env['website'].get_current_website()
```

Comment:
Hello @tivisse 
When i was developing this task page search was not working in master. So i have fixed it in my branch. During my development someone make fix for it at https://github.com/odoo/odoo/commit/a98d60cb11ceee31a6bd9d3734d0e4b565e5d677  **which was not working either** so i reverted that commit in branch and fix it again.

Now during that another fix has been made to fix previous broken fix   https://github.com/odoo/odoo/commit/273a15ea7d6e23e14b939c518fd1d7cfcd0cff94 

should i remove it or keep it ???